docs/microbit_micropython_api.rst

```diff
@@ -157,8 +163,13 @@ Except in the case of the pins marked **V2**, which offers the following API:
 
 pin_logo::
 
-    # returns boolean for logo touch pin
+    # returns a boolean for logo touch pin
     pin_logo.is_touched()
+    # returns a boolean if the logo was pressed since the last time
+    # this method was called
+    pin_logo.was_touched()
+    # returns the running total of touches, and resets this counter to zero
+    pin.get_touches()
```

Comment:
Should be `pin_logo.get_touches()`.  Probably also the example below should be changed to `pin_logo.set_touch_mode(value)`.

Reply:
These methods are also available for pins 0, 1 and 2, so we are using `pin.` here as it's been done for  `pin.is_touched()`.

Reply:
But, this section/paragraph of the docs is all about `pin_logo` (see line 158/164 above).  And the code snippets just above are for `pin_logo.is_touched()` and `pin_logo.was_touched()`.

So, it should be `pin_logo.get_touches()` and `pin_logo.set_touch_mode()` for consistency within this paragraph.

Reply:
Oh yes, you are right, I didn't notice in my last message that this was replicated in a pin_logo-only section.
Updated in https://github.com/bbcmicrobit/micropython/compare/4043fc290df1ef1cf281c8211995eca66e27a63f..1fbb7581cf90b6976b183cca855c2cc71989744f.